beangulp/__init__.py

```diff
@@ -17,6 +17,7 @@
 from beangulp import file
 from beangulp import identify
 from beangulp import importer
+from beangulp.importer import Importer
```

Comment:
I love it. We should make all public symbols on the top-level eventually, so that all importer code for the most part ever just uses "import beangulp"